wtf.py

```diff
@@ -322,22 +323,23 @@ def run(self):
     all_fixed += sum( fixed[k] for k in actions )
     if args.verbose>=1:
         if problems_seen>0 or args.verbose>=2:
-            print("%s:" % fname, file=stderr)
-            if actions.trail_space:
-                print("\t%s %d lines with trailing space" % ('CHOPPED' if actions.trail_space=='fix' else 'SAW', seen.trail_space), file=stderr)
-            if actions.eof_blanks:
-                print("\t%s %d blank lines at EOF" % ('CHOPPED' if actions.eof_blanks=='fix' else 'SAW', seen.eof_blanks), file=stderr)
-            if actions.eof_newl:
-                print("\t%s newline at EOF" % ('ADDED' if actions.eof_newl=='fix' and fixed.eof_newl else 'SAW MISSING' if seen.eof_newl else 'no change to'), file=stderr)
-            if actions.coerce_eol:
-                print("\t%s %d line endings which didn't match %s%s" % ('CHANGED' if actions.coerce_eol[0]=='fix' else 'SAW', seen.coerce_eol,
-                    eol_val2name[fp.eol_value], ' from first line' if actions.coerce_eol[1]=='first' else ''), file=stderr)
-            if actions.tab_space_mix:
-                print("\t%s %d lines with mixed tabs/spaces" % ('CHANGED' if actions.tab_space_mix=='fix' else 'WARNED ABOUT' if actions.tab_space_mix=='report' else 'SAW', seen.tab_space_mix), file=stderr)
-            if actions.change_tabs is not None:
-                print("\tCHANGED tabs to %d spaces on %d lines" % (actions.change_tabs, fixed.change_tabs if fixed.change_tabs > 0 else seen.change_tabs), file=stderr)
-            if actions.change_spaces is not None:
-                print("\tCHANGED %d spaces to tabs on %d lines" % (actions.change_spaces, fixed.change_spaces if fixed.change_spaces > 0 else seen.change_spaces), file=stderr)
+            with redirect_stdout(stderr):
```

Comment:
Hmmmm… will this work correctly with `wtf` used as a pipe, e.g. `cat file.txt | wtf > file_wtf.txt`? Even on Windows? :grimacing:

Reply:
I want to say yes, but let me test it first.
In the cases of failures we would see the `CHANGED` messages on the `file_wtf.txt` right?

Reply:
just tested on windows with `type .editorconfig | python wtf.py > file_wtf.txt` and it works as expected. `file_wtf.txt` gets generated with its expected (fixed) contents, and the `stderr` messages are out on the command prompt.